packages/amplify-category-auth/src/provider-utils/awscloudformation/utils/amplify-meta-updaters.ts

```diff
@@ -1,69 +1,87 @@
 import * as path from 'path';
-import { JSONUtilities, pathManager, $TSAny } from 'amplify-cli-core';
+import {
+  JSONUtilities, pathManager, $TSAny, $TSContext,
+} from 'amplify-cli-core';
 import { hostedUIProviders } from '../assets/string-maps';
 import { AuthParameters } from '../import/types';
 
+type FrontEndConfig = {
+  socialProviders: string[];
+  usernameAttributes: string[];
+  signupAttributes: string[];
+  passwordProtectionSettings: {
+    passwordPolicyMinLength: number | undefined;
+    passwordPolicyCharacters: string[];
+  };
+  mfaConfiguration: string | undefined;
+  mfaTypes: string[];
+  verificationMechanisms: string[];
+}
+
 /**
  * Factory function that returns a function that updates Amplify meta files after adding auth resource assets
  *
  * refactored from commands/enable.js
  * @param context The amplify context
  * @param resultMetadata The metadata from the service selection prompt
+ * @param resultMetadata.service the service
+ * @param resultMetadata.providerName the provider
  */
-export const getPostAddAuthMetaUpdater =
-  (context: any, resultMetadata: { service: string; providerName: string }) =>
-  (resourceName: string): string => {
-    const options: any = {
-      service: resultMetadata.service,
-      providerPlugin: resultMetadata.providerName,
-    };
-    const parametersJSONPath = path.join(context.amplify.pathManager.getBackendDirPath(), 'auth', resourceName, 'build', 'parameters.json');
-    const authParameters = JSONUtilities.readJson<AuthParameters>(parametersJSONPath)!;
-
-    if (authParameters.dependsOn) {
-      options.dependsOn = authParameters.dependsOn;
-    }
+export const getPostAddAuthMetaUpdater = (context: $TSContext,
+  resultMetadata: {
+    service: string;
+    providerName: string
+  }) => (resourceName: string): string => {
```

Comment:
nit: I prefer having parameters with their own indentation block for readability, but not a blocker
```suggestion
export const getPostAddAuthMetaUpdater = (
  context: $TSContext,
  resultMetadata: {
    service: string;
    providerName: string
  },
) => (resourceName: string): string => {
```